packages/css/css.0.1.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "CSS parser and printer"
```

Comment:
```suggestion
synopsis: "CSS parser and printer"
description: """
OCaml-css is an OCaml library to parse and print CSS. It can also expand namespaces and perform computations on property values.

Parsing uses [Angstrom](https://github.com/inhabitedtype/angstrom/) to dynamically choose the parser when encountering a property declaration.

The parser can be extended by defining additional properties."""
```